### DISCUSSION TITLE
能否考虑支持115网盘的支持？

### DISCUSSION THREAD

**Discussion by izscc** [AlistGo/alist#235] [Ideas]
能否考虑支持115网盘的支持？ 115网盘的空间大 非常适合挂alist 对船友俱乐部而言 资源获取也方便

**Comment by fengguowudi** [comment]
这个没api吧

**Comment by xhofe** [comment]
同 #241 
需要携带cookie的只能通过服务器中转，而绝大多数用户的服务器（国内）带宽都不大，所以暂不考虑。

**Reply by tyokantei** [reply]
我不知道kodi有人开发了115的插件，是可以播放的，虽然原盘也是卡顿，但是4K 还是可以的，115只要能用webdav，用nplayer播放就行了，不知道行不行，如果可行的话，我可以赞助大佬开发一个

**Comment by zixiu13** [comment]
CloudDrive可以把115挂载到本地，alist能否借鉴下，也支持115

**Reply by edisi0n** [reply]
115非会员，网页限速50k，这个速度没有太大意义吧

**Comment by q4232134** [comment]
https://github.com/gaoyb7/115drive-webdav
有大佬开发了115的webdav，可以和alist适配，亲测有用

**Reply by libixi123** [reply]
大哥，怎么搭配法，麻烦麻烦给个指导呗，拜托了

**Reply by q4232134** [reply]
额，aist支持webdav，然后那个项目可以把115转换成webdav，这不就直接对接了么

**Reply by libixi123** [reply]
win11成功了，Win10怎么也不成，因为没有步骤指导，还是不知道哪里出错了。

**Reply by q4232134** [reply]
emmmmm，跟系统没啥关系啊，我用的docker弄的，基于arm64的centeros。
![image](https://user-images.githubusercontent.com/8042695/191874408-64b9b5ec-4b6a-4245-9e24-53af1217be3f.png)
